backend/typescript/middlewares/validators/schedulingValidators.ts

```diff
@@ -57,9 +60,25 @@ export const createSchedulingDtoValidator = async (
       .status(400)
       .send(getApiValidationError("volunteerNeeded", "boolean"));
   }
-  if (!validatePrimitive(req.body.frequency, "string")) {
+  if (
+    req.body.volunteerTime &&
+    !validatePrimitive(req.body.volunteerTime, "string")
+  ) {
+    return res
+      .status(400)
+      .send(getApiValidationError("volunteerTime", "string"));
+  }
+  if (!Object.values(Frequency).includes(req.body.status)) {
```

Comment:
should this be frequency?
```suggestion
  if (!Object.values(Frequency).includes(req.body.frequency)) {
```

Reply:
Yes thank you for catching that!